graphql/schema/wrappers.go

```diff
@@ -2345,6 +2348,36 @@ func (fd *fieldDefinition) IsID() bool {
 	return isID(fd.fieldDef)
 }
 
+func (fd *fieldDefinition) GetDefaultValue(action string) interface{} {
+	if fd.fieldDef == nil {
+		return nil
+	}
+	return getDefaultValue(fd.fieldDef, action)
+}
+
+func getDefaultValue(fd *ast.FieldDefinition, action string) interface{} {
+	dir := fd.Directives.ForName(defaultDirective)
+	if dir == nil {
+		return nil
+	}
+	arg := dir.Arguments.ForName(action)
+	if arg == nil {
+		return nil
+	}
+	value := arg.Value.Children.ForName("value")
+	if value == nil {
+		return nil
+	}
+	if value.Raw == "$now" {
+		if flag.Lookup("test.v") == nil {
+			return time.Now().Format(time.RFC3339)
+		} else {
+			return "2000-01-01T00:00:00.00Z"
+		}
```

Comment:
This should fix the lint error.

```suggestion
		}
		return "2000-01-01T00:00:00.00Z"
		
```